### PR TITLE
Dutch localization

### PR DESCRIPTION
**Pull request by JappaWakka**

### CODE REVIEW

**Comment by dsbilling** [comment]
Marked as a draft, click "ready for review" when it is.

File: resources/lang/nl/auth.php [hunk 1]
```diff
@@ -3,5 +3,5 @@
 return [
     'failed'   => 'Deze combinatie van e-mailadres en wachtwoord is niet geldig.',
     'password' => 'Het opgegeven wachtwoord is onjuist.',
-    'throttle' => 'Te veel mislukte loginpogingen. Probeer het over :seconds seconden nogmaals.',
+    'throttle' => 'Teveel mislukte aanmeldpogingen. Probeer het over nog eens over :seconds seconden.',
```
Comment (dsbilling) [inline resources/lang/nl/auth.php:6]: Changes in this file will be overwritten by sync with [Laravel-Lang/Lang](https://github.com/Laravel-Lang/lang). Consider contributing to that repo too.

File: resources/lang/nl/validation.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php
```
Comment (dsbilling) [inline resources/lang/nl/validation.php:1]: Changes in this file will be overwritten by sync with [Laravel-Lang/Lang](https://github.com/Laravel-Lang/lang). Consider contributing to that repo too.

**Review by dsbilling — APPROVED** [review]

File: resources/lang/nl/auth.php (hunk 1 above)
Comment (JappaWakka) [reply resources/lang/nl/auth.php:6]: Good idea, will do.

File: resources/lang/nl/validation.php (hunk 2 above)
Comment (JappaWakka) [reply resources/lang/nl/validation.php:1]: Good idea, will do.

**Pull request merged**